processing/src/HpsEventFile.cxx

```diff
@@ -38,4 +38,4 @@ bool HpsEventFile::nextEvent() {
   intree_->GetEntry(entry_++);
   
   return true;
-} 
+}
```

Comment:
I don't think we need to change the whitespace here in this PR.

Reply:
I don't honestly don't know why it keeps saying I fiddled with the whitespace. This is not intentional. I changed it back I think from the previous whitespace. I promise we'll adopt a convention when I talk with Sarah.